src/lib/testing-utils.ts

```diff
@@ -7,13 +8,17 @@ export type MemoryFS = {
 }
 
 export function writeToFS(cwd: string, vfs: MemoryFS) {
-  Object.entries(vfs).forEach(([fileOrDirName, fileContentOrDir]) => {
-    const subPath = Path.join(cwd, fileOrDirName)
+  Object.entries(vfs).forEach(([path, fileContentOrDir]) => {
+    const absolutePath = Path.join(cwd, path)
 
     if (typeof fileContentOrDir === 'string') {
-      FS.write(subPath, fileContentOrDir)
+      if (!fs.existsSync(Path.dirname(absolutePath))) {
```

Comment:
code comment? anything `todo`?

Reply:
Reverted that as it turned out to be useless